src/pretalx/submission/models/submission.py

```diff
@@ -981,14 +981,33 @@ def score_categories(self):
     @cached_property
     def active_resources(self):
         return self.resources.filter(
-            models.Q(  # either the resource exists
-                ~models.Q(resource="")
-                & models.Q(resource__isnull=False)
-                & ~models.Q(resource="None")
+            models.Q(
+                models.Q(  # either the resource exists
+                    ~models.Q(resource="")
+                    & models.Q(resource__isnull=False)
+                    & ~models.Q(resource="None")
+                )
+                | models.Q(  # or the link exists
+                    models.Q(link__isnull=False) & ~models.Q(link="")
+                )
+            )
+            & models.Q(is_public=True)
```

Comment:
I think I'd prefer this as a plain `, is_public=True`, and I feel like we should rename `active_resources` to `public_resources` here – or possibly keep `active_resources` as the overarching property, and then make `active_resources` and `private_resources` simple filters on `active_resources`, with the upside of less code duplication aswell.